src/Uno.UI/UI/Xaml/Controls/Control/Control.cs

```diff
@@ -91,10 +91,10 @@ private void OnTemplateChanged(DependencyPropertyChangedEventArgs e)
 		internal void SetUpdateControlTemplate(bool forceUpdate = false)
 		{
 			if (
```

Comment:
If we rollback the whole change, it means that we will have to use the legacy mode in all our applications... Invoke the `ApplyTemplate` just before the `OnLoaded` on Android did not do the trick?

Reply:
Yes, if you need lazy template loading for the new activities you'll need to set the UseLegacyLazyApplyTemplate to true, until #119 is addressed.

Reply:
Yeah I thought this flag would make our features incompatible, but it will break the `DataGrid`. I assume that no matter the name we can live with it for now :)